modules/services/fundraising/main.tf

```diff
@@ -65,13 +65,13 @@ resource "aws_lb_listener_rule" "fundraising" {
 
 // Create ECS task definition
 resource "aws_ecs_task_definition" "fundraising" {
```

Comment:
The task definition should be renamed also, isn't?
`resource "aws_ecs_task_definition" "fundraising"` -> `resource "aws_ecs_task_definition" "membership"`